deployment/terraform/live/vars.tf

```diff
@@ -30,21 +30,26 @@ variable "vpc_cidr_block" {
   default = "10.0.0.0/24"
 }
 
-variable "public_cidr_block" {
+variable "public_cidr_block_a" {
   type    = "string"
-  default = "10.0.0.0/25"
+  default = "10.0.0.0/26"
 }
 
-variable "private_cidr_block" {
+variable "public_cidr_block_b" {
```

Comment:
I thought you needed two private subnets in separate AZs for the RDS, not two public ones?

Reply:
Technically its the former or the latter, but in our case it makes more sense to have the second private one